src/mixer/playermanager.cpp

```diff
@@ -34,11 +32,8 @@ const int kNumberOfAnalyzerThreads = math_max(1, QThread::idealThreadCount() / 2
 
 } // anonymous namespace
 
-//static
 QAtomicPointer<ControlProxy> PlayerManager::m_pCOPNumDecks;
-//static
 QAtomicPointer<ControlProxy> PlayerManager::m_pCOPNumSamplers;
-//static
 QAtomicPointer<ControlProxy> PlayerManager::m_pCOPNumPreviewDecks;
```

Comment:
Why did you remove these annotations? Please do not change code that is irrelevant to the PR!